docker/2diirunner-with-packages/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM maurolepore/2diirunner:1.3.0
+MAINTAINER "Mauro Lepore" maurolepore@gmail.com
+
+RUN apt-get update && apt-get install -y pandoc
+
+# This repository has binaries not just for Windows and Mac, but also for Linux
+RUN echo "options(repos = c(CRAN = 'https://packagemanager.rstudio.com/all/__linux__/bionic/latest'))" >> /root/.Rprofile \
```

Comment:
In my experience, just setting the default repo in options isn't enough... you must also configure the HTTP UserAgent properly. You can do that with something like
```
echo 'options(HTTPUserAgent = sprintf("R/%s R (%s)", getRversion(), paste(getRversion(), R.version$platform, R.version$arch, R.version$os)))' >> ~/.Rprofile
```
You can also run RStudio's test script to tell if it is setup properly, like...
```
Rscript -e 'source("https://docs.rstudio.com/rspm/admin/check-user-agent.R")'
```
if it's setup properly, you'll get a message like `Success! Your user agent is correctly configured.`